### DISCUSSION TITLE
Quest reward in party.

### DISCUSSION THREAD

**Discussion by ZerausDG** [FTBTeam/FTB-Mods-Issues#944] [Q&A]
Hello, sorry for my English, I use translator, is it normal that when I do quest in party only 1 person of the party can claim the reward and the others can't?

I changed the settings to see if that was the problem but it was still the same

**Comment by desht** [comment]
This is controlled by the `Team Reward` property of the reward.  When true, the reward is a team reward, meaning it can be claimed only once per team.  When false, everyone in the team can claim it.

When it's set to Default, like here:

![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/736831/13475fb6-a90d-48f9-9c94-3e5fd8d82f44)

... it means to use the `Reward "Team Reward"` global setting, which can be edited with "Edit File" (click the cog wheel lower right)

![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/736831/78ad15a5-32ac-41a9-955d-9c7da8ed842e)

Here it's set to False, which means everyone in the team can collect the reward.
